var/docs/docker.md

```diff
@@ -0,0 +1,66 @@
+# Docker
+
+We bundle a simple docker that can be used for test purposes.  Either pull it from docker hub or build it.
+
+## Build the docker:
+
+    docker build -t kimai/kimai2 .
+
+## Run the docker
+
+    docker run -ti -p 8001:8001 --name kimai2 --rm kimai/kimai2
+
+The you can then access the site on http://127.0.0.1:8001 or on Mac you may need to use the docker IP:
+
+    docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kimai2
```

Comment:
This does not work on my Mac, only `docker-machine ip default` gives me the right solution. If you don't mind, I will fix the typos and add a section about Mac directly in your repo

Reply:
@kevinpapst Please go ahead, I don't have a mac to test on :)